umbral/utils.py

```diff
@@ -78,9 +81,9 @@ def unsafe_hash_to_point(curve, data, label=None):
         ibytes = i.to_bytes(4, byteorder='big')
         digest = hashes.Hash(hashes.SHA256(), backend=default_backend())
         digest.update(label + ibytes + data)
-        hash = digest.finalize()
+        hash_digest = digest.finalize()
 
-        compressed02 = b"\x02"+hash
+        compressed02 = b"\x02" + hash_digest
```

Comment:
Wow, look at the improvement in this line.  Rarely will you see a better example of the importance of good names.